### ISSUE TITLE
missing #ifdef guard in the header file

### ISSUE THREAD

**Issue by noisemaker00** [adafruit/Adafruit_ADS1X15#41]
The `Adafruit_ADS1015.h` header file does not have any guard to check multiple imports.
Please add something like:
```
#ifndef ADAFRUIT_ADS1015_H
#define ADAFRUIT_ADS1015_H
...original file...
#endif
```

**Comment by siddacious** [comment]
@noisemaker00 If you want to make a PR to add the guard we'd appreciate it!

**Comment by caternuson** [comment]
fixed by #47
